### DISCUSSION TITLE
[Suggestion] Allow pet owners and administrators to untame pets

### DISCUSSION THREAD

**Discussion by DelboyDylan** [GriefPrevention/GriefPrevention#1229] [Ideas]
When searching for a horse it often happens that you end up claiming a bunch of them that you have no interest in. Then have to repeatedly use /GivePet when someone else comes along, and wants to take one. When these are in remote locations, this is rather bothersome to have to go out there each and every time. Therefore, being able to untame unwanted pets, so they can be tamed by others at a later date would be handy.

In addition, players can conceivably grief a server by taming every tamable animal they find, thus preventing others from getting pets. Therefore having this as an option for administrators (who are using /IgnoreClaims) would also be helpful, so batches of animals can be untamed, allowing other players get them at a later date, if they so choose.

**Comment by LadyViviann** [comment]
This would also be helpful since I started a racetrack on our server with a barn full of animals. Doing /givepet on each one before and after every race is very time consuming. /givepet public would be nice. I know I can do /containeraccess public but that allows them into all the chests, etc. And I'm not sure it will let them ride the pet outside of the claim. Only the barn is claimed, not the racetrack.
